template/__tests__/App.test.tsx

```diff
@@ -3,11 +3,9 @@
  */
 
 import React from 'react';
-import ReactTestRenderer from 'react-test-renderer';
+import { render } from "@testing-library/react-native";
 import App from '../App';
 
 test('renders correctly', async () => {
```

Comment:
This test does not contain async code, so we can safely remove async keyword
```suggestion
test('renders correctly', () => {
```